src/screens/HomeScreen.tsx

```diff
@@ -21,6 +22,26 @@ export default function HomeScreen() {
   const isValidLocation = (tree: Tree) =>
     tree.location && tree.location.latitude && tree.location.longitude;
 
+  const getCoordinates = (mapboxJSON: Mapbox) => {
+    const { features } = mapboxJSON;
+    return features.map((feature: Feature) => {
+      const coordinates = feature.geometry.coordinates.map(
+        (coord: number[]) => ({
+          latitude: coord[1],
+          longitude: coord[0],
+        }),
+      );
+      return (
+        <Polygon
+          coordinates={coordinates}
```

Comment:
can you add a unique key prop here? there's a warning. maybe the id provided in mapBox